acceptance/tests/loader/func4x_loadable_from_modules.rb

```diff
@@ -80,4 +80,9 @@ class helloworld {
     assert_match(/^Generated, 1 => 10, 2 => 20, 3 => 30$/, stdout, "Generated the wrong content")
   end
 
+  teardown do
```

Comment:
I'd favor reverting e6e0ae0. It *seems* like that should just work, but I might be missing something.